### ISSUE TITLE
Dutch: Quotes around title do not include comma before year in `\parencite`

### ISSUE THREAD

**Issue by LukeSerne** [plk/biblatex-apa#220]
**Description**
The localisation of APA7 by Tilburg University [uses American-style quotes](https://libguides.uvt.nl/apa-handleiding/naslagwerken#s-lg-box-wrapper-18778652), that also incorporate trailing punctuation. I have tried to reproduce the example from their guide as the minimal working example below. Unfortunately, the trailing comma in the `\parencite` is not put inside the quotes. This is observed, even if `\usepackage[style=american, autopunct]{csquotes}` is added to the document preamble.

**MWE**
```latex
\documentclass{report}

\usepackage[dutch]{babel}
\usepackage[style=apa, backend=biber]{biblatex}

\begin{filecontents}[overwrite]{test.bib}
@inbook{Participatieladder2022,
    title = {Participatieladder},
    date = {2022-04-21},
    booktitle = {Wikipedia},
    url = {https://nl.wikipedia.org/w/index.php?title=Participatieladder&oldid=34040095},
}
\end{filecontents}
\addbibresource{test.bib}

\begin{document}
\parencite{Participatieladder2022}
\printbibliography
\end{document}
```

**Observed output**
(“Participatieladder”, 2022)

**Expected output**
(“Participatieladder,” 2022)

**Comment by plk** [comment]
This is strange - I would have expected loading csquotes with the right options would have fixed this. @moewew - any ideas?

**Comment by moewew** [comment]
Are we sure people actually want this behaviour in a Dutch document? I'm always a bit wary of third-party websites interpreting APA style and the like. (Granted, this is *translated* APA style, so it is slightly different. Especially if there is no local counterpart to APA that could step in. In Germany I believe the DGPs is seen as giving the "official APA translation".)

American-style quotation marks are already uncommon enough in parts of the English-speaking world, so I'm really surprised that people would actively want that to spill over to Dutch as well. Would Tilburg University recommend moving punctuation inside quotation marks in all cases or only in citations?

**Comment by plk** [comment]
Yes, I'm also not sure this should be in the style but it's odd it can't seem to be changed with the csquotes functionality. I'm not sure how the general `biblatex` handling of this interacts with csquotes when it's used.

**Comment by moewew** [comment]
I haven't had time to check, but I think we need `biblatex`'s `\uspunctuation` here (cf. https://github.com/plk/biblatex/blob/dev/tex/latex/biblatex/lbx/american.lbx). `csquotes` alone can't do that. The code to move the `biblatex`-generated punctuation inside quotation marks is pretty complex and needs support on the `biblatex` end as well.

**Comment by plk** [comment]
Hmm, then it's worth, @LukeSerne, to check if this really is necessary for APA conformance in Dutch. It does seem odd that they would want US punctuation rules here which isn't really part of the APA style per se.

**Comment by moewew** [comment]
On first glance `\uspcunctuation` appears to work in this instance.

```latex
\documentclass{article}

\usepackage[dutch]{babel}
\usepackage[style=apa, backend=biber]{biblatex}

\DefineBibliographyExtras{dutch}{\uspunctuation}

\begin{filecontents}{\jobname.bib}
@inbook{Participatieladder2022,
  title     = {Participatieladder},
  date      = {2022-04-21},
  booktitle = {Wikipedia},
  url       = {https://nl.wikipedia.org/w/index.php?title=Participatieladder&oldid=34040095},
}
\end{filecontents}
\addbibresource{\jobname.bib}

\begin{document}
\parencite{Participatieladder2022}
\printbibliography
\end{document}
```

> (“Participatieladder,” 2022)

My doubts about whether this is actually necessary remain.

**Comment by LukeSerne** [comment]
I am personally not too happy with the way Tilburg University has decided to handle this. They have come up with [their own APA7 localisation](https://libguides.uvt.nl/apa-handleiding), which is subtly different from other localisations ([here](https://www.scribbr.nl/wp-content/uploads/2020/11/De-Nederlandse-APA-regels-Handleiding-7e-editie-Scribbr.pdf) is one from Scribbr (a company), [here](https://www.auteursrechten.nl/files/auteursrechten/2021-07/De%20APA-richtlijnen%20uitgelegd%20-%203e%20editie.pdf) is one from SURF (a collaboration between universities of applied sciences) - also used by [Radboud University](https://libguides.ru.nl/apa7/nederlandse-conventies)). Other universities seem to only require reports written in English (and thus there is no need for a Dutch localisation of APA7) or they just mention that there ["is no standard"](https://libguides.library.uu.nl/citeren/apa) for Dutch APA7 and list the aforementioned documents. There's also [this blog](http://theapateam.blogspot.com/p/welkom.html) that gives additional advice on using APA7 in Dutch, in line with the localisation by SURF. From a cursory look at the localisations by SURF and Scribbr, it seems they are more similar to each other than to the localisation by Tilburg University. Throughout the SURF manual, there are examples showing the quote going before trailing punctuation (for instance on page 55).

Since I had to deal with Tilburg University's requirements, I tried conforming to their rules. However, their localisation is far from perfect. Just looking through their examples, you'll find many obvious typo's, or examples that they clearly forgot to translate from English. It seems they just translated (some of) the examples included with the APA7 manual, and occasionally made a mistake. It's impossible to write a good system based on this, and I took some liberties in interpreting what is a typo and what isn't.

In this specific case, there I only found one instance where there is a quote-character followed by punctuation. In that case, the punctuation is moved inside the quote character. They never explicitly say to use American punctuation rules, however, they do like to use the Oxford comma in author and editor lists. Nevertheless, it doesn't seem impossible that this is just a typo / improper localisation of the example provided by the APA7 manual.

I don't mind considering this a typo and not changing the behaviour of biblatex-apa, since Tilburg University's rules seem to be "the odd one out". Since many more people are using the SURF and Scribbr manuals, it would be better to tailor the localisation to those manuals. I am happy to know that `\DefineBibliographyExtras{dutch}{\uspunctuation}` works to change the behaviour on a per-document basis.
